protobuf/proto-types/eventer.proto

```diff
@@ -1,17 +1,17 @@
 syntax = "proto2";
 
-package wrapper;
+package eventer;
 
 import "net_msg.proto";
 import "net_conn.proto";
 import "addrman.proto";
 import "mempool.proto";
 import "validation.proto";
 
-message Wrapper {
+message Eventer {
```

Comment:
```suggestion
message EventMsg {
```

I don't think [Eventer](https://dictionary.cambridge.org/dictionary/english/eventer) is a proper English word describing this message. I would just use `EventMsg` here.

Reply:
Makes sense!!👍